bukkit/src/main/java/dev/jsinco/brewery/bukkit/effect/event/DrunkEventExecutor.java

```diff
@@ -17,62 +43,66 @@
 
 public class DrunkEventExecutor {
 
-    private Map<UUID, List<EventStep>> onJoinExecutions = new HashMap<>();
+    private final Map<UUID, List<EventStep>> onJoinExecutions = new HashMap<>();
+
+    public DrunkEventExecutor() {
+        EventStepRegistry registry = TheBrewingProject.getInstance().getEventStepRegistry();
+        registry.register(ChickenNamedEvent.class, o -> new ChickenNamedExecutable());
```

Comment:
And this could be

```suggestion
        registry.register(BreweryKey.fromString("chicken"), o -> new ChickenNamedExecutable());
```
(And so on)